themes/ddbasic/sass/components/header.scss

```diff
@@ -641,9 +641,10 @@
           top: $_base-top-value + $_value + $search-form-extended-height;
         }
 
-        #{$_selector}.has-second-level-menu & {
-          top: $_base-top-value + $_value + $second-level-menu-height;
-        }
+        //This style misplaced the search input on herbib site.
+        // #{$_selector}.has-second-level-menu & {
+        //   top: $_base-top-value + $_value + $second-level-menu-height;
+        // }
```

Comment:
@mihaelad Would this affect search bar placement on other sites that make use of ddbasic theme?

Reply:
@mihaelad This change breaks the display of extended search bar then main menu has secondary level menu items.

![easyopac-1350](https://user-images.githubusercontent.com/800338/130406547-99ea2786-ead6-455c-a830-406ad93e2694.PNG)

Please review your solution.

Reply:
Yes, you right! It will be better to add in the css editor.